Streaming with ACL required arguments

y-luis-rojo opened the pull request:
az: error: argument _command_package: invalid choice: site
az webapp log tail: error: (--resource-group --name | --ids) are required

azurecla commented:
Hi __@cdavid__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
@lizap Hi Liza - we have a [small but important] contribution in the public azure-content repo.  When you have a minute, could you review quickly and LMK if it's OK for merge?

cdavid commented:
Hi

This is ntdev\catalda :) I notified Liza via corp email as well about this
change. What are the next steps?

Thanks,
Catalin

On Tue, Jun 2, 2015 at 2:50 PM, Bryan Lamos <notifications@github.com>
wrote:

> @lizap <https://github.com/lizap> Hi Liza - we have a [small but
> important] contribution in the public azure-content repo. When you have a
> minute, could you review quickly and LMK if it's OK for merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-content/pull/3669#issuecomment-108109130>.
>



-- 
*************************************
Catalin David
M.Sc. Smart Systems 2012
B.Sc. Computer Science 2010
Jacobs University Bremen
*************************************

lizap commented:
Please go ahead and merge this. Catalin is definitely correct with this change.

Thanks, Liza

BryanLa commented:
Thanks David.

I just need Liza to sign off on the change, since she represents the RemoteApp service for our pillar, either in the PR or via email.

Lisa: this is just a change in your “RemoteApp Service Limits” article (https://azure.microsoft.com/en-us/documentation/articles/remoteapp-servicelimits/ ), to reduce collection limit “Published apps per collection” from 5000 to 100, per David.

When you get a second just LMK if it’s OK to merge.

Thanks,
Bryan

From: cdavid [mailto:notifications@github.com]
Sent: Tuesday, June 2, 2015 3:46 PM
To: Azure/azure-content
Cc: Bryan Lamos
Subject: Re: [azure-content] Change published apps per collection limit (#3669)

Hi

This is ntdev\catalda :) I notified Liza via corp email as well about this
change. What are the next steps?

Thanks,
Catalin

On Tue, Jun 2, 2015 at 2:50 PM, Bryan Lamos <notifications@github.com<mailto:notifications@github.com>>
wrote:

> @lizap <https://github.com/lizap> Hi Liza - we have a [small but
> important] contribution in the public azure-content repo. When you have a
> minute, could you review quickly and LMK if it's OK for merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-content/pull/3669#issuecomment-108109130>.
>



--
*************************************
Catalin David
M.Sc. Smart Systems 2012
B.Sc. Computer Science 2010
Jacobs University Bremen
*************************************


—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/3669#issuecomment-108120549>.

y-luis-rojo committed:
Streaming with ACL required arguments

az: error: argument _command_package: invalid choice: site
az webapp log tail: error: (--resource-group --name | --ids) are required
